src/open_inwoner/utils/context_processors.py

```diff
@@ -82,6 +82,9 @@ def settings(request):
         "cookie_link_text": config.cookie_link_text,
         "cookie_link_url": config.cookie_link_url,
         "extra_css": config.extra_css,
+        "custom_javascript_url": config.custom_javascript.url
+        if config.custom_javascript
```

Comment:
I think the confirmation should also be required to actually show the file:
```suggestion
        if config.custom_javascript and config.custom_javascript_confirmed
```